tests/unit/s2n_tls13_handshake_test.c

```diff
@@ -383,8 +383,8 @@ int main(int argc, char **argv)
         EXPECT_NOT_NULL(cert_chain = malloc(S2N_MAX_TEST_PEM_SIZE));
         EXPECT_NOT_NULL(private_key = malloc(S2N_MAX_TEST_PEM_SIZE));
 
-        EXPECT_SUCCESS(s2n_read_test_pem(S2N_ECDSA_P384_PKCS1_CERT_CHAIN, cert_chain, S2N_MAX_TEST_PEM_SIZE));
-        EXPECT_SUCCESS(s2n_read_test_pem(S2N_ECDSA_P384_PKCS1_KEY, private_key, S2N_MAX_TEST_PEM_SIZE));
+        EXPECT_SUCCESS(s2n_read_test_pem(S2N_RSA_2048_PKCS1_CERT_CHAIN, cert_chain, S2N_MAX_TEST_PEM_SIZE));
```

Comment:
I think it's better to keep S2N_ECDSA_P384_PKCS1_* here. S2N_RSA_2048_PKCS1_CERT_CHAIN could be used when RSA-PSS-RSAE is implemented.

Reply:
Originally I was using RSA because ECDSA's max version was set to less than TLS 1.3, but it seems that the update in https://github.com/awslabs/s2n/pull/1553 fixed it, so I'll switch it back to ECDSA.